coalib/bearlib/__init__.py

```diff
@@ -41,16 +54,24 @@ def deprecate_settings(**depr_args):
     def _deprecate_decorator(func):
 
         def wrapping_function(*args, **kwargs):
-            for arg, deprecated_arg in depr_args.items():
+            for arg, _deprecated_arg in depr_args.items():
```

Comment:
`_deprecated_arg` isn't actually the deprectated arg which is why your names are so confusing and clashing. It's rather a tuple or the arg right? Make a better name. Clashes usually suggest bad naming, underscores doesn't solve the real problem :)